tests/test_rooms.py

```diff
@@ -0,0 +1,146 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+from __future__ import annotations
+
+import asyncio
+from datetime import datetime
+
+import pytest
+from ypy_websocket.yutils import write_var_uint
+
+from jupyter_collaboration.loaders import FileLoader
+from jupyter_collaboration.rooms import DocumentRoom
+from jupyter_collaboration.utils import RoomMessages
+
+from .utils import FakeContentsManager, FakeEventLogger, FakeFileIDManager
+
+
+@pytest.mark.asyncio
+async def test_should_initialize_document_room_without_store():
+    id = "test-id"
+    content = "test"
+    paths = {id: "test.txt"}
+    cm = FakeContentsManager({"content": content})
+    loader = FileLoader(
+        id,
+        FakeFileIDManager(paths),
+        cm,
+        poll_interval=0.1,
+    )
```

Comment:
I'm wondering if this should be a fixture or a factory to avoid too many copy paste.

> Fully optional - happy to merge as is.

Reply:
I looked at it again. Even though it seems like code duplication, it is not. Each test uses a different argument, and we need to create and parameterize multiple fixtures.

Let's merge the PR, and I'll work on creating a proper set of helpers and fixtures for testing.